recipe/meta.yaml

```diff
@@ -111,39 +108,18 @@ outputs:
         - gym.envs.classic_control
 
   - name: gym-mujoco
-    build:
-      # mujoco-py cannot be packaged for license reasons, see
-      # https://github.com/openai/mujoco-py#install-mujoco
-      skip: true
     requirements:
       host:
         - python
       run:
         - python
         - {{ pin_subpackage('gym', exact=True) }}
-        - mujoco-py >=1.50,<2.0
+        - mujoco-python
         - imageio
     test:
       imports:
         - gym.envs.mujoco
 
-  - name: gym-robotics
```

Comment:
Has this (sub-)package been removed upstream?

Reply:
Yes, there are a few links in the PR description on this.

Reply:
Ok, sounds good